pythonmod/pythonmod.c

```diff
@@ -310,6 +321,9 @@ int pythonmod_init(struct module_env* env, int id)
 #endif
       SWIG_init();
       mainthr = PyEval_SaveThread();
+
+      /* XXX: register callback to unwind Python at exit */
+      atexit(pythonmod_atexit);
```

Comment:
Removing the XXX; I don't suppose there is anything more to do here.
```suggestion
      /* register callback to unwind Python at exit */
      atexit(pythonmod_atexit);
```

Reply:
> Removing the XXX; I don't suppose there is anything more to do here.

Agreed. Thanks